Feature/DESIGN 116 lists

TrebBrennan opened the pull request:
review, vert-list needs work

TrebBrennan committed:
Start of list work

Note hot fix for article in main

TrebBrennan committed:
List view HTML+CSS

Commit:
Add lists guidance section

TrebBrennan committed:
Added .non-list class

.non-list

TrebBrennan committed:
Responsive & .hbs file

TrebBrennan committed:
Fix for Datetime attributes

TrebBrennan committed:
Visual changes to accommodate small lists

TrebBrennan committed:
Few tweaks

TrebBrennan committed:
More list types

review, vert-list needs work

maxious commented:
Doesn't seem to work just yet:

![screen shot 2016-07-04 at 8 55 33 am](https://cloud.githubusercontent.com/assets/81432/16548051/1d554716-41c5-11e6-9607-2240bc9c5830.png)

Commit:
Adds some content changes

@TrebBrennan
Changes previously in `feature/116-lists-content` branch.

Can discuss.

Commit:
Refactor of Horizontal list item styles

Commit:
Adjust List content to match markup changes

Commit:
Refactor vertical view for Lists

Commit:
Refactor small view for Lists

Commit:
Update List content

Commit:
Add example horizontal list item that should have an image but doesn't

Commit:
Add a minimum height to headings on Horizontal list items

Commit:
Add notes to horizontal List about omiting img

Commit:
Change CSS Gulp build so that autoprefix runs after sass

Commit:
Add variants of the Vertical list with 3 and 4 columns

maxious committed:
fix list example

maxious committed:
fix IE9 comment
